waku/v2/protocol/waku_rln_relay/rlnzerokit/waku_rln_relay_utils.nim

```diff
@@ -0,0 +1,841 @@
+{.push raises: [Defect].}
```

Comment:
There is no need for a duplicate of waku_rln_relay_utils, you can use the existing one

Reply:
The RLN construct definition should be moved to the rln.nim module and then all the imports of rln.nim module should be managed under a compiler flag.